core/domain/exp_services.py

```diff
@@ -792,6 +793,15 @@ def _save_exploration(committer_id, exploration, commit_message, change_list):
                 'which is too old. Please reload the page and try again.'
                 % (exploration_model.version, exploration.version))
 
+    if feconf.ENABLE_STRING_CLASSIFIER:
```

Comment:
OK, no worries. But next time please double-check stuff before you comment (or, for that matter, push a set of commits for review)! Note that the "Files Changed" tab is a good place to look at and self-review your code prior to asking for a review.